docs/gitbook/chain-management/cascading-deployment.md

```diff
@@ -0,0 +1,85 @@
+
+## The Root Node
+
+The root node contains a copy of the deployed manifest file and a write join to each of your high level chains. The high level chains in turn contain a write join to their children etc. This write join allows your root node to forward updates to the manifest to your chains should it receive an update. These updates will be forwarded by interbit down the line to child chains until all network updates have been applied.
+
+A configuration file must describe an acyclic network for cascading deployment to work. The network shape must be a tree.
+
+### Root Covenant
+
+The root node covenant must be a part of all covenants that wish to enforce cascading deployment. If the root node covenant is not applied to a smart contract the chain will not automatically apply updates and deployments and system upgrades must be managed in another way.
+
+## The Interbit CLI
+
+The interbit CLI can generate and deploy manifest files and watch the chains for updates to the root manifest configuration. Running `interbit-cli build` on a valid configuration will generate a deployable manifest file. Running `interbit-cli deploy` will take a manifest file and deploy a node based on the instructions in the manifest and passed through command line options.
+
+This will spin up a set of watchers that will apply any updates to the chain when a new manifest is deployed.
+
+### Root Watcher
+
+The root watcher watches the root node for CLI level update commands. If there is a difference in a newly deployed manifest compared to the previous manifest the root watcher will apply some of the changes.
+
+The root watcher
+ - deploys and applies covenants
+ - creates new chains
+ - removes chains that are no longer in the manifest
+
+### Chain Watchers
+
+One chain watcher is created for each chain that is running on a deployed node. Each chain watcher
+ - applies join changes for that chain
+
+# Permission to deploy
+Entirely governed by the permissions of the machine you are on to access the root chain.  Deployment is always done by the machines global hypervisor, rather than the temporary hypervisor that runs in dev mode
+
+# Dynamic vs. Static Chains
+
+Static chains are chains that make up the supporting structure of your system. They are chains that do not change often and support other chains. These static chains are defined in the Interbit manifest file and are managed by the root node and interbit-cli. Static chains are created during the deployment of your system.
+
+Dynamic chains are chains that are created during the operation of your system. These chains may be ephemeral chains or last for a long period. These chains are created by smart contracts on chains already running within your system. The chain or process that creates a dynamic chain is responsible for udpating and managing the dynamic chain's lifecycle.
+
```

Comment:
Move the discussion of dynamic and static chains up as this is important. Specifically, make it clearer that dynamic chains are not specified in the manifest